Eatery Blue/UI/EateryCard/EateryExpandableCardMenuCategoryView.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  EateryExpandableCardMenuCategoryView.swift
+//  Eatery Blue
+//
+//  Created by Antoinette Marie Torres on 9/27/23.
+//
+
+import EateryModel
+import SnapKit
+import UIKit
+
+class EateryExpandableCardMenuCategoryView: UIView {
+    
+    // MARK: - Properties
+    
+    private let categoryNameLabel = UILabel()
+    private let foodItemLabel = UILabel()
+    
+    // MARK: - init
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        setupCategoryNameLabel()
+        setupFoodItemLabel()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - configure
+    
+    func configure(menuCategory: MenuCategory) {
+        var foodItemString = ""
+        menuCategory.items.forEach { item in
+            foodItemString.append("\(item.name)\n")
+        }
+        
+        foodItemLabel.text = String(foodItemString.dropLast(1))
+        categoryNameLabel.text = menuCategory.category
+    }
+    
+    // MARK: - Set Up Views
+    
+    private func setupCategoryNameLabel() {
+        categoryNameLabel.textColor = UIColor.Eatery.black
+        categoryNameLabel.font = UIFont.preferredFont(for: .title3, weight: .semibold)
```

Comment:
You used just `.preferredFont` in the following files and I checked other parts of the codebase and I think it uses just the static var so I'd probably change this to `.preferredFont` for consistency.